www/caddy/Portfile

```diff
@@ -3,32 +3,34 @@
 PortSystem          1.0
 PortGroup           golang 1.0
 
-go.setup            github.com/mholt/caddy 0.11.4 v
+# Caddy uses its own tool (xcaddy) to build the caddy executable with version
+# information baked in
+go.setup            github.com/caddyserver/xcaddy 0.1.8 v
+
+name                caddy
+version             2.3.0
+revision            0
 categories          www
 platforms           darwin
 supported_archs     i386 x86_64
```

Comment:
Let's remove `supported_archs` so that the golang portgroup does the right thing automatically on whichever architecture.

Reply:
Done